### PR TITLE
fix: fix error messages

### PR DESCRIPTION
**Pull request by EduardoGiacomini**
### Overview
This PR translate the error messages return by apollo graphql.

### Pull request type
- [ ] Bug fixes;
- [x] New feature;
- [ ] Refactoring;
- [ ] Documentation;
- [ ] Other: __________________

### Current behaviour
The error messages uses only english.

### New behaviour
Now, the error messages are translated to Portuguese

*Sorry if I have done English mistakes, I make this PR so fast.*

### CODE REVIEW

File: src/permissions/errors.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  DEFAULT: 'Ops, estamos com problemas em nossos servidores. Por favor, tente novamente',
+
+  // Generics
+  BAD_USER_INPUT: 'Os dados informados estão incorretos',
+  NOT_ALLOWED: 'Usuário não autorizado',
+  UNAUTHENTICATED: 'Usuário não autenticado',
+
+  // Specific
+  CAN_NOT_UPDATE_OTHER_USER: 'Você não pode alterar a senha de outros usuários',
+  CAN_NOT_UPDATE_OTHER_USER_PASSWORD: 'Você não pode alterar a senha de outros usuários',
```
Comment (F-Camuso) [inline src/permissions/errors.ts:12]: Same message

File: src/permissions/rules.ts [hunk 2]
```diff
@@ -1,26 +1,54 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
 import { rule } from 'graphql-shield';
+import ERRORS from './errors';
 
-export const notAllawed = rule({ cache: 'contextual' })(async (parent, args, ctx, info) => false);
+export const notAlloawed = rule({ cache: 'contextual' })(async (parent, args, ctx, info) => false);
```
Comment (F-Camuso) [inline src/permissions/rules.ts:5]: should be notAllowed

**Review by F-Camuso — CHANGES_REQUESTED** [review]
lgmt, just small fixes 🚀

**Comment by EduardoGiacomini** [comment]
@F-Camuso, maybe it's correct now. 🤪

**Review by F-Camuso — APPROVED** [review]
Great job! Expensive player 😎

**Review by SantiagoMateus — APPROVED** [review]

**Pull request merged**